molgenis-data-mapper/src/main/java/org/molgenis/data/mapper/service/impl/MappingServiceImpl.java

```diff
@@ -173,7 +176,6 @@ public String applyMappings(MappingTarget mappingTarget, String entityName)
 	public String applyMappings(MappingTarget mappingTarget, String entityName, boolean addSourceAttribute)
 	{
 		EntityType targetMetaData = EntityType.newInstance(mappingTarget.getTarget(), DEEP_COPY_ATTRS, attrMetaFactory);
-		targetMetaData.setPackage(null);
```

Comment:
Because sometimes the given target entity is already existing and in another package. Without setting this setPackage to null, the package name is added twice for some reason.